docker/runtime/nodejs/Dockerfile.8.opt

```diff
@@ -0,0 +1,21 @@
+FROM node:8 AS build-env
+
+RUN apt-get update &&  apt-get install git
+
+ADD lib/helper.js kubeless_rt/lib/
+ADD kubeless.js kubeless_rt/
+ADD package.json kubeless_rt/
+
+WORKDIR kubeless_rt/
+
+RUN npm install
+
+
+FROM gcr.io/distroless/nodejs
```

Comment:
is there a label here ? I wonder how often they push those images and how they keep them up to date ?

Reply:
I found the following issue on distroless https://github.com/GoogleContainerTools/distroless/issues/166

Reply:
not sure what's the difference but this is the list of tags:
https://console.cloud.google.com/gcr/images/distroless/GLOBAL/nodejs?gcrImageListsize=50

Reply:
so I think we should break up the runtimes into their own repo (maybe a single repo for all runtime). Then had automation in that runtime repo to build all images including the base ones (like a base distroless image).

The problem is that this base image will be updated without us knowing.

Reply:
see https://github.com/kubeless/kubeless/issues/762

Reply:
This looks good to me until we split runtimes in a different repository so we have a bit more of flexibility as @sebgoa comments. Can you select a tag from the list so we don't get caught by an issue using `latest`?

Reply:
I tried to open the list of tags link. Even when I log in I cannot see anything. The content is being loaded forever. Can you list some of the tags?

Reply:
The current digest of the latest image is `sha256:91b207c7278667472dcd08d8e137ed98c99a3b92120f6a7ec977fc3f63323848` you can use that one.